File: ReportingApplication/ReportingApplication/Models/Recipent.cs
namespace ReportingApplication.Models
{
    public class Recipent(string userId, string recipentName)
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public string UserId { get; set; } = userId;
        public string RecipentName { get; set; } = recipentName;
        public List<Ticket> Tickets { get; set; } = new List<Ticket>();
    }
}

File: ReportingApplicationFrontend/src/app/models/recipent.ts
export class Recipent{
    userId?: string | null;
    recipentName?: string | null;
    recipentColor?: string | null;

    constructor(_userId?: string | null, _recipentName?: string | null, _recipentColor?: string | null){
        this.userId = _userId;
        this.recipentName = _recipentName;
        this.recipentColor = _recipentColor;
    }
}